client/types.go

```diff
@@ -427,7 +427,12 @@ type Signup struct {
 }
 
 type Tenant struct {
-	Signup Signup `json:"signup"`
+	Signup              Signup  `json:"signup"`
+	FromEmail           *string `json:"fromEmail,omitempty"`
+	SupportEmail        *string `json:"supportEmail,omitempty"`
```

Comment:
Account struct - which is part of Signup - already has the supportEmail https://github.com/3scale/3scale-porta-go-client/blob/60dca7443cc76dafa02ddeb90855f38a5b2349c7/client/types.go#L358

What's the difference between Signup.Account.SupportEmail and Tenant.SupportEmail?

Reply:
@MStokluska thank you for review. Should we remove Tenant.SupportEmail and use Signup.Account.SupportEmail ?

Reply:
Done. Thank you for comment.

Reply:
What is the reason for SupportEmail to be within signup and other emails at a higher level?

Reply:
any suggestions?

Reply:
thank you, discussed, it was addressed already.